CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Contributing to ESEm
+
+Firstly, thank you for considering contributing to ESEm! As a small unfunded package any contributions are welcome, small or large :-)
+
+## Questions
+If you have a general question regarding installing / using / understanding ESEm then please feel free to use our [Discussions](https://github.com/duncanwp/ESEm/discussions) page where someone will be able to help.
+
+## Issues
+If you have a problem with ESEm and it isn't working as documented or is producing unexpected errors then please check in the [Issues](https://github.com/duncanwp/ESEm/issues) page to see if anyone else has reported a similar problem. 
+Feel free to comment on associated issues so that we know who it's affecting. 
+
+If there isn't an appropriate existing issue, please create one and be as descriptive as possible about what you expected to happen and what actually happened. Please also explain what you've tried to fix it.
+
+## Enhancements and other constributions 
+We welcome suggestions for improvements and (even better!) pull requests implementing such improvements. 
+We hope the code is quite readable such that extensions and improvements are easy to implement and welcome first-time pull-requests. 
+Some useful guides to making a pull-request can be found at http://makeapullrequest.com/ and http://www.firsttimersonly.com/.
+
+We are proud to have excellent test coverage for the code base and will request that any new contributions ensure similar coverage.
+
+## Roadmap
+While we don't have an explicit roadmap, it is intended that ESEm remains a fairly light-weight wrapper around other packages which themselves implement e.g. emulation routines.
+With this in mind we have broadly defined areas we would welcome contributions, and those we would consider out of scope for ESEm below.
+
+### Welcome contributions
+ - We're very keen to include more diverse or advanced sampling techniques
+ - Further wrappers for emulation/regression techniques (ideally using existing dependencies, such as Keras or scikit-learn)
+ - Improvements to documentation / code comments
+ - Other examples / notebooks
+ - Other data pre-processors
+
+### Contributions likely to be rejected
+ - Implementation of emulation techniques within ESEm
```

Comment:
I get what you mean here, but the line might get a bit blurry in some instances? Would suggest adding "from scratch" just to make it clearer. i.e., we don't want someone building a whole new GP/RF infrastructure within ESEm